docs/root/intro/arch_overview/other_features/connection_limiting.rst

```diff
@@ -0,0 +1,7 @@
+.. _arch_overview_connection_limit:
```

Comment:
see https://www.envoyproxy.io/docs/envoy/latest/configuration/listeners/runtime#config-listeners-runtime. Instead of adding a new "other features" section can you actually merge "connection limiting" into the listener docs and make sure you cover both features and ways of limiting connections?

Reply:
I am not quite sure about what you said. I just followed the `local_ratelimit` docs organization.
Do you prefer another way to do that?

Reply:
Please do the following:
1) Do not add this here, it's not really an "other feature"
2) Add a connection limiting doc on the listener section of the arch overview
3) Reference all of the various features in that section.

I think this will read better.

/wait

Reply:
Added `connection_limit_filter.rst` into the listener docs and referenced the `connection_limiting.rst` in it.
Please take a look, thanks!